api/specs.proto

```diff
@@ -208,6 +208,16 @@ message ContainerSpec {
 	// will be exposed to the container.
 	repeated SecretReference secrets = 12;
 
+	// Hosts allow additional entries to be specified in /etc/hosts
+	// that associates IP addresses with hostnames.
+	// Detailed documentation is available in:
+	// http://man7.org/linux/man-pages/man5/hosts.5.html
+	// The format of the Hosts here could be:
+	// <hostname>:<ip> (separated by `:`)
+	// <hostname>=<ip> (separated by `=`)
+	// <hostname> <ip> (separated by ` `)
```

Comment:
These examples don't seem to match the http://man7.org/linux/man-pages/man5/hosts.5.html 

```
For each host a single line should be present with the following information:

              IP_address canonical_hostname [aliases...]

Fields of the entry are separated by any number of blanks and/or tab
characters.  Text from a "#" character until the end of the line is a
comment, and is ignored.  Host names may contain only alphanumeric
characters, minus signs ("-"), and periods (".").  They must begin
with an alphabetic character and end with an alphanumeric character.
Optional aliases provide for name changes, alternate spellings,
shorter hostnames, or generic hostnames (for example, localhost).
```